deploy/template.yaml

```diff
@@ -1299,6 +1320,155 @@ Resources:
               Bool:
                 "aws:SecureTransport": true
 
+  #
+  # Metrics and Alarms
+  #
+
+  # API Gateway Alarms
+
+  # 4xx Status Code
+  ApiGateway4xxStatusAlarm:
+    Condition: IsNotDevelopment
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmName: !Sub "${AWS::StackName}-ApiGateway4xxStatusAlarm"
+      AlarmDescription: "Trigger the alarm if errorThreshold exceeds 1% with the minimum of 10 invocations in 2 out of the last 5 minutes"
+      ActionsEnabled: false  # disabled until we're ready to invoke alarm actions
+      AlarmActions:
+        - !Sub "{{resolve:ssm:/${AlertingStackName}/SNS/HighAlertNotificationTopic/ARN}}"
```

Comment:
Nice, are these the topics we'd been mentioning in the slack PR? The priority based topics we could use for Dev Platform stuff too?

Reply:
Yes that's right we have two alert based topics depending on severity, it was agreed to have a high alert SNS topic for P1/P2 alerts and a low alert SNS topic for P3/P4 type alerts such as high latency as per Slack discussion with @alex9smith [here](https://gds.slack.com/archives/C011Y5SAY3U/p1669909920448659 and given we now manage our own alerting template we can add additional SNS topic pubs/subs if and when needed.